### PR TITLE
Add in fixes to control definitions

### PR DESCRIPTION
**Pull request by mlysaght2017**

### CODE REVIEW

File: services/common-controls.yaml [hunk 1]
```diff
@@ -70,16 +70,18 @@ controls:
...
           - tlp_green
           - tlp_amber
           - tlp_red
       - id: CCC.C02.TR02
         text: |
-          Admin users can verify and audit encryption status for stored data at rest,
-          including verification of key management processes.
+          Admin users can verify and audit encryption status for
+          stored data at rest, including verification of key
+          management processes.
```
Comment (sarahecraddock) [inline services/common-controls.yaml:84]: Do we want to make the same change here to remove admin users from description?

**Review by sarahecraddock — APPROVED** [review]
Looks good @mlysaght2017, just one suggestion that I spotted where the admin user language is used in a similar way that it might be worth updating to align with the others you updated 😊

File: services/common-controls.yaml (hunk 1 above)
Comment (mlysaght2017) [reply services/common-controls.yaml]: Done now!

**Comment by mlysaght2017** [comment]
@damienjburks - can you please approve and merge - thanks

**Review by damienjburks — APPROVED** [review]

**Pull request merged**